lib/redis/time_series/client.rb

```diff
@@ -57,21 +51,29 @@ def redis
       #
       # @param client [Redis] a Redis client
       # @return [Redis]
-      def redis=(client)
-        @redis = client
+      def redis=(conn)
+        $redis = conn # = TimeSeries::ConnectionPoolProxy.proxy_if_needed(conn)
```

Comment:
We'll have to avoid using a global variable here as that has the potential to conflict with or pollute existing apps.

Reply:
Fixes #79 though!